app/assets/javascripts/app/views/header_view.js

```diff
@@ -7,36 +7,27 @@ app.views.Header = app.views.Base.extend({
   className : "dark-header",
 
   events : {
-    "click ul.dropdown li:first-child" : "toggleDropdown",
+    "click ul.nav.navbar-nav li:first-child" : "toggleDropdown",
     "focusin #q": "toggleSearchActive",
     "focusout #q": "toggleSearchActive"
   },
 
   initialize : function(options) {
-    $(document.body).click($.proxy(this.hideDropdown, this));
     return this;
   },
 
   menuElement : function() {
-    return this.$("ul.dropdown");
+    return this.$("ul.nav.navbar-nav");
```

Comment:
@dimaursu : [this JS file](https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/widgets/header.js) is embedded with the page. It instanciates [this](https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/widgets/notifications-badge.js) and [this](https://github.com/diaspora/diaspora/blob/develop/app/assets/javascripts/widgets/notifications.js). I already corrected the code a bit to make the number of notifications to be displayed but I don't manage to make the dropdown work. :/

This code will have to be changed a lot. But I suck in JS, so if I do it alone, it'll take a lots of time :p